content/en/news/releases/1.11.x/announcing-1.11/upgrade-notes/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Upgrade Notes
+description: Important changes to consider when upgrading to Istio 1.11.0.
+weight: 20
+---
+
+When you upgrade from Istio 1.10.0 to Istio 1.11.0, you need to consider the changes on this page.
```

Comment:
@Monkeyanator @jacob-delgado @therealmitchconnors We call out upgrading from 1.10 to 1.11, but we now support 1.9 to 1.11 as well. Should this paragraph be updated to also talk about 1.9 and or point to the 1.10 back talking about changes from 1.9? Or is it understood that the user should have to see both?